webapp/src/components/conference/conference.test.tsx

```diff
@@ -42,7 +42,12 @@ describe('Conference', () => {
         post: basePost,
         jwt: null,
         actions,
-        currentUserId: 'test'
+        currentUser: {
+            id: 'mockId',
+            first_name: 'First',
+            last_name: 'Last',
+            username: 'firstLast'
+        }
```

Comment:
CI is not running on this PR. It should catch things like the fact that this is not a full `UserProfile`. Not necessarily a very useful thing to catch, but this signaled to me that CI is not running

Reply:
@ayusht2810 Can you run the CI commands locally and fix anything that comes up?